### ISSUE TITLE
windowcoverings_set capability is invalid

### ISSUE THREAD

**Issue by jghaanstra** [athombv/apps-sdk-issues#24]
I'm trying to add the windowcoverings_set capability to a device (Shelly 2 Roller Shutter). When a user pairs a Shelly 2 as Roller Shutter the Homey.addDevice({}) is called but the device is not added with an error 'invalid capability'. My app.json looks like this:

```
"class": "windowcoverings",
"capabilities": [
	"windowcoverings_state",
  "windowcoverings_set",
	"measure_power"
],
```
And related code in device.js for this device is:

`this.registerCapabilityListener('windowcoverings_set', this.onCapabilityWindowcoveringsSet.bind(this));`

```
onCapabilityWindowcoveringsSet(value, opts, callback) {
    var position = value * 100;
    util.sendCommand('/roller/0?go=to_pos&roller_pos='+ position, this.getSetting('address'), this.getSetting('username'), this.getSetting('password'));
    callback(null, value);
  }
```

It looks to me there is something wrong with either the API documentation or Homey core where the capability windowcoverings_set returns as an invalid capability.

**Comment by WeeJeWel** [comment]
Have you tried pairing without `windowcoverings_set`? Does that work?

**Comment by jghaanstra** [comment]
Yup, setting the roller shutte with a percentage has been recently added to the Shelly 2 device with a firmware update. So I tried adding the windowscoverings_set capability to the existing Shelly 2 Roller Shutter driver so Homey can control the Shelly 2 with percentages as well. Before adding this capability the driver worked perfectly but after adding the capability it returns the invalid capability error. I have checked my own code numerous of times but I'm pretty sure there is something fishy going on with the windowcoverings_set capability.

It was tested by a user on Homey 1.5.13 by the way.
